packages/sql-mapper/lib/entity.js

```diff
@@ -382,7 +382,7 @@ async function buildEntity (db, sql, log, table, queries, autoTimestamp, schema,
   /* istanbul ignore next */
   function checkSQLitePrimaryKey (constraint) {
     if (db.isSQLite) {
-      const validTypes = ['integer', 'uuid', 'serial']
+      const validTypes = ['integer', 'uuid', 'serial', 'varchar']
       const pkType = fields[constraint.column_name].sqlType.toLowerCase()
       if (!validTypes.includes(pkType)) {
         throw new Error(`Invalid Primary Key type. Expected "integer", found "${pkType}"`)
```

Comment:
```suggestion
        throw new Error(`Invalid "${pkType}" Primary Key type. Only ${validTypes.join(', ')} are supported.`)
```
This error message should be fixed since sqlite allows to use not only the "integer" type.

Reply:
@ivan-tymoshenko thank you for the review! i have adjusted this